crates/matrix-sdk/Cargo.toml

```diff
@@ -92,11 +92,13 @@ ruma = { workspace = true, features = ["rand", "unstable-msc2448", "unstable-msc
 serde = { workspace = true }
 serde_html_form = { workspace = true }
 serde_json = { workspace = true }
+serde_urlencoded = "0.7.1"
```

Comment:
We already depend on `serde_html_form` which is a fork of `serde_urlencoded`, can you please use that one instead?